src/pages/[...params].js

```diff
@@ -14,6 +14,8 @@ const Fallback = () => {
       router.push('/login/');
     }
 
+    delete query.params;
```

Comment:
```suggestion
    // Remove the "params" query parameter added by Next 
    // (which holds the current path, based on `[...params]` in the filename)
    // because it causes infinite redirects as it keeps adding this query parameter to the iframe's URL
    delete query.params;
```